components/Carousel.vue

```diff
@@ -0,0 +1,188 @@
+<script>
+import BadgeState from '@/components/BadgeState';
+
+export default {
+  components: { BadgeState },
+  name:       'Carousel',
+  props:      {
+    sliders: {
+      type:    Array,
+      default: null
+    }
+  },
+  data() {
+    return {
+      slider:          this.sliders,
+      activeItemId:    0
+    };
+  },
+
+  computed: {
+
+    trackStyle() {
+      return `transform: translateX(-${ this.activeItemId * 100 / this.slider.length }%); width: calc(${ 60 * this.slider.length }%)`;
+    },
+
+  },
+
+  methods: {
+    scrollSlide(i) {
+      this.activeItemId = i;
+      setTimeout(() => {
+        if (i <= 1) {
+          this.$refs.slide[this.slider.length - 1].style.left = '-93%';
+          this.$refs.slide[0].style.left = '7%';
+        } else {
+          this.$refs.slide[this.slider.length - 1].style.left = '7%';
+          this.$refs.slide[0].style.left = '107%';
+        }
+      }, 400);
+    },
+  }
+};
+
+</script>
+
+<template>
+  <div class="slider">
+    <div id="slide-track" ref="slider" style="transition: 1s ease-in-out" :style="trackStyle" class="slide-track">
+      <div
```

Comment:
We are missing one of the most important aspect of this feature, which is the link to the chart 😅

Reply:
In the card, nothing is mentioned about links and we don't know the actual data yet so I will keep it as it is.